### ISSUE TITLE
This code is causing websockets to fail

### ISSUE THREAD

**Issue by deleolajide** [igniterealtime/ofmeet-openfire-plugin#25]
This code in OfMeetPlugin.java

````
		Security.addProvider( new OfMeetSaslProvider() );
		SASLAuthentication.addSupportedMechanism( OfMeetSaslServer.MECHANISM_NAME );
````

is causing websockets to fail. We should either refine and approve the PR (https://github.com/igniterealtime/Openfire/pull/835)  made to fix it  or remove this code that adds OFMEET as an external SASL authentication mechanism.

I use the websockets instead of http-bind.

**Comment by guusdk** [comment]
https://github.com/igniterealtime/Openfire/pull/835 has now been merged, and will be available in the next releases of Openfire (4.1.6 and 4.2.0). Can this issue be closed?

**Comment by deleolajide** [comment]
yes indeed
